apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -266,50 +120,24 @@ def remove_router_interface(self, context, router_id, interface_info):
     def create_floatingip(self, context, floatingip):
         res = super(ApicL3ServicePlugin, self).create_floatingip(
             context, floatingip)
-        port_id = floatingip.get('floatingip', {}).get('port_id')
-        self._notify_port_update(port_id)
-        if res:
-            res['status'] = self._update_floatingip_status(context, res['id'])
+        fip_context = api.FipContext(res, original=floatingip)
```

Comment:
you can get both these values from the FIP dict returned by the superclass method. So I think you only need to set 'current' to 'res'.